docs/common_text/vadr_flu_segments.md

```diff
@@ -0,0 +1,10 @@
+??? task "`vadr_flu_segments`"
```

Comment:
add ==_for flu in TheiaCoV_FASTA_== to the end of this title to match format of other things